### PR TITLE
STREAMS-1440 - Refactor Streams helm chart

### PR DESCRIPTION
**Pull request by cremond**
Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x ] You have verified the technical accuracy of your change
* [x ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [x] Does not have conflicts with the base branch
* [x] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [x] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [x] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [x] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [x] Does not contain typos, grammatical errors, etc.
* [x] Does not expose any sensitive information
* [x] Passes the Markdown linter rules (automated via CI check)
* [x] Does not contain broken links

### CODE REVIEW

File: content/en/docs/install/_index.md [hunk 1]
```diff
@@ -84,14 +84,14 @@ The MariaDB database is automatically installed by default on your K8s cluster w
...
 To disable the MariaDB installation, you can either:
 
-* Edit the `values.yaml` file and set the `mariadb.enabled` entry as follow:
+* Edit the `values.yaml` file and set the `embeddedMariadb.enabled` entry as follow:
 
 ```yaml
-mariadb:
-  enabled: false
+embeddedMariadb:
+enabled: false
```
Comment (benjaminranc) [inline content/en/docs/install/_index.md:91]:
```suggestion
  enabled: false
```

**Review by benjaminranc — CHANGES_REQUESTED** [review]

**Comment by cremond** [comment]
Hello @allandenis38 this pull request has been reviewed by @benjaminranc and is ready for your review. It is about a change of a few flags (including the embeddedMariadb embeddedKafka changes when it comes to _ our code_) due to a massive refactoring of helm. Watch out for the handling of k8s.yourdomain.tld which has been modified to hopefully make it less confusing. The jira ticket is STREAMS-1440.

**Review by allandenis38 — APPROVED** [review]

**Comment by allandenis38** [comment]
@lbadenhop PR ready for your review
Preview: https://deploy-preview-55--streams-open-docs.netlify.app/

File: content/en/docs/install/_index.md [hunk 2]
```diff
@@ -133,10 +133,10 @@ export DB_STREAMS_USER="streams"
 mysql -h "${DB_HOST}" -P "${DB_PORT}" -u "${DB_USER}" -p -e "GRANT SELECT, INSERT, UPDATE, DELETE ON ${DB_NAME}.* TO ${DB_STREAMS_USER} REQUIRE SSL;"
 ```
 
-Then you must provide information to the Streams installation. You should edit the `values.yaml` file and set the `externalMariadb` entry as follow:
+Then you must provide information to the Streams installation. You should edit the `values.yaml` file and set the `externalizedMariadb` entry as follow:
```
Comment (lbadenhop) [inline content/en/docs/install/_index.md:136]:
```suggestion
Then you must provide information to the Streams installation. You should edit the `values.yaml` file and set the `externalizedMariadb` entry as follows:
```

File: content/en/docs/install/_index.md [hunk 3]
```diff
@@ -171,7 +171,7 @@ According to your choice, you must:
     export NAMESPACE="my-namespace"
     kubectl create secret generic streams-database-secret --from-file=CA_PEM=ca.pem -n ${NAMESPACE}
     ```
-    * Set the [Helm parameters](#helm-parameters) `externalMariadb.tls.twoWay` to `false`.
+    * Set the [Helm parameters](#helm-parameters) `externalizedMariadb.tls.twoWay` to `false`.
 
 * For Two-Way TLS:
```
Comment (lbadenhop) [inline content/en/docs/install/_index.md:176]:
```suggestion
* For two-way TLS:
```

File: content/en/docs/install/_index.md [hunk 4]
```diff
@@ -181,7 +181,7 @@ According to your choice, you must:
     ```
 
 * For no TLS:
-    * Set the [Helm parameters](#helm-parameters) `externalMariadb.tls.enabled` to `false`.
+    * Set the [Helm parameters](#helm-parameters) `externalizedMariadb.tls.enabled` to `false`.
 
 You can follow the official documentation provided by Mariadb [Certificate Creation with OpenSSL](https://mariadb.com/kb/en/certificate-creation-with-openssl/) to generate self-signed certificate. Make sure to set the Common Name correctly.
```
Comment (lbadenhop) [inline content/en/docs/install/_index.md:186]:
```suggestion
See the official documentation provided by MariaDB [Certificate Creation with OpenSSL](https://mariadb.com/kb/en/certificate-creation-with-openssl/) to generate self-signed certificates. Make sure to set the Common Name correctly.
```

File: content/en/docs/install/_index.md [hunk 5]
```diff
@@ -423,7 +430,14 @@ helm install "${HELM_RELEASE_NAME}" . \
 ```
 
 {{< alert title="Note" >}}
-The default configuration only accepts incoming HTTP/HTTPS requests to `k8s.yourdomain.tld`.
+You will be required to specify a hostname. If you do not have one yet, you can use any temporary value and edit it later.
+For this entire documentation we are using k8s.yourdomain.tld as an _example_ value.
```
Comment (lbadenhop) [inline content/en/docs/install/_index.md:434]:
```suggestion
Throughout this documentation, we are using k8s.yourdomain.tld as an _example_ value.
```

**Review by lbadenhop — COMMENTED** [review]
See suggestions

**Pull request merged**